addons/hr/views/res_config_settings_views.xml

```diff
@@ -43,6 +43,9 @@
                         <setting help="Enrich employee profiles with skills and resumes" id="enrich_employee_setting">
                             <field name="module_hr_skills"/>
                         </setting>
+                        <setting id="home_working_setting" help="Display remote working settings for each employee and dedicated reports. Presence icons will be updated with remote working location.">
```

Comment:
We talk about Remote Work, not Remote Working. Please adapt help field. 
Thanks,